### PR TITLE
include 'pkg/' in rake/clean's CLOBBER array

### LINKED ISSUE #3676: Add `pkg` to Rake's CLOBBER FileList
Bundler's gem_tasks are quite helpful. But the first thing that needs added to a new gem's Rakefile is frequently `require 'rake/clean'` and `CLOBBER.include 'pkg'`

Since 'pkg' is hardcoded into the `build` task, might it also be handy to pre-populate the `CLOBBER` FileList with 'pkg' as well?

### PR DESCRIPTION
**Pull request by jasonkarns**
closes #3676

### CODE REVIEW

File: spec/runtime/gem_tasks_spec.rb [hunk 1]
```diff
@@ -32,4 +34,9 @@
     expect(tasks & expected_tasks).to eq(expected_tasks)
     expect(exitstatus).to eq(0) if exitstatus
   end
+
+  it "includes 'pkg' in rake/clean's CLOBBER" do
+    require 'bundler/gem_tasks'
```
Comment (segiddins) [inline spec/runtime/gem_tasks_spec.rb]: Double quotes.

Comment (jasonkarns) [inline spec/runtime/gem_tasks_spec.rb]: fixed

**Comment by segiddins** [comment]
@homu r+

Thanks @jasonkarns !

**Comment by jasonkarns** [comment]
Any estimate on next release?

**Pull request merged**